jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/JPADelegatingClassFileTransformer.java

```diff
@@ -42,7 +43,10 @@ public ByteBuffer transform(ClassLoader classLoader, String className, Protectio
                     // run as security privileged action
                     @Override
                     public ByteBuffer run() {
-
+                        if(Configuration.ignoreClassTransform(className)) {
+                            // indicate that class change was not made as workaround for https://hibernate.atlassian.net/browse/HHH-16572
```

Comment:
Will create a separate issue tracker for removing the call to check Configuration.ignoreClassTransform(className) if/when HHH-16572 is fixed (when possible).